gnovm/pkg/gnopkgfetch/gnopkgfetch.go

```diff
@@ -0,0 +1,130 @@
+package gnopkgfetch
```

Comment:
Please move it to `gnovm/cmd/gno`, it shouldn't be in this place in the `gnovm`, or probably even exported

It's super use-case specific functionality that doesn't warrant a package

Reply:
7bbd0da7e26a6f1ac1a65422b132f694248167f5